Apps/CesiumViewer/CesiumViewer.js

```diff
@@ -33,7 +37,10 @@ define([
      * 'source' : 'file.czml',  // The relative URL of the CZML file to load at startup.
      * 'stats'  : true,         // Enable the FPS performance display.
      * 'theme'  : 'lighter',    // Use the dark-text-on-light-background theme.
-     * 'scene3DOnly' : false    // Enable 3D only mode
+     * 'scene3DOnly' : false,    // Enable 3D only mode
+     * 'saveCamera' : false,    // Saves the location and orientation of the camera in the query string for reload
+     * 'location' : string/lon,lat/lon,lat,height // Fly to this location on load.
```

Comment:
Thinking more about this maybe a default height of zero would not be used, so the string could be:
```
location=lon,lat,height,[heading],[pitch],[roll]  // or camera=
```
What do you think?